index.html

```diff
@@ -12325,10 +12439,11 @@ <h3>States and Properties</h3>
 					</ul>
 				</li>
 				</ul>
+			<p>If the author specfies a WAI-ARIA property on a role where that property is prohibited, the user agent SHOULD treat the property as if it were allowed. </p>
```

Comment:
Perhaps we could put an editor's note after this statement to further discourage authors from ignoring the prohibition? Candidate text:

> Note: At the present time, user agents have no obligation to ignore prohibited states and properties. However, this may change in a future version of ARIA in order to ensure interoperability.

Reply:
We agreed at the meeting to remove the text from the author errors section I believe. Could this go after line 498 instead?